desc/optimize/optimizer.py

```diff
@@ -411,97 +216,90 @@ def optimize(  # noqa: C901 - FIXME: simplify this
         return result
 
 
+def _parse_method(method):
+    """Split string into wrapper and method parts."""
+    wrapper = None
+    submethod = method
+    for key in Optimizer._wrappers[1:]:
```

Comment:
this parse_method, it seems to exist so we can do like `opt = Optimizer(method="prox-lsq-exact")` or something like that?

is there a test showing that use case?

Reply:
Yeah I modified some of the examples to use the proximal wrapper explicitly, and it's also the default for `eq.optimize`